rioxarray/xarray_plugin.py

```diff
@@ -0,0 +1,42 @@
+import os.path
+
+import xarray as xr
+
+from . import _io
+
+CAN_OPEN_EXTS = {"geotif", "geotiff", "j2k", "jp2", "tif", "tiff", "vrt"}
+
+
+class RasterioBackend(xr.backends.common.BackendEntrypoint):
+    def open_dataset(
+        self,
+        filename_or_obj,
+        drop_variables=None,
+        mask_and_scale=False,
```

Comment:
You could probably mask_and_scale to `True` since this is a entrypoint. It would make it more consistent with other engines using `xarray.open_dataset`.

Reply:
Done. And what about `masked`?

Reply:
`masked` was added before `mask_and_scale` and is there for backwards compatibility. It is mostly useful for loading in datasets that don't require scaling but masking is desired as it has simpler logic for loading in data. Being `False` by default allows the user to set `mask_and_scale=False` without also having to set `masked=False`.